CONTRIBUTING.md

```diff
@@ -1,59 +1,58 @@
-# Contributing guidelines
-
-We thank you in advance :thumbsup: :tada: for taking the time to contribute, whether with *code* or with *ideas*, to the NeIC SDA project.
+# Contributing Guidelines
 
+We thank you in advance 👍 🎉 for taking the time to contribute, whether with *code* or with *ideas*, to the NeIC SDA project.
 
 ## Did you find a bug?
 
-* Ensure that the bug was not already reported by [searching under Issues].
-
-* If you're unable to find an (open) issue addressing the problem, [open a new one]. Be sure to prefix the issue title with **[BUG]** and to include:
+- Ensure that the bug was not already reported by [searching under Issues].
 
-  - a *clear* description,
-  - as much relevant information as possible, and
-  - a *code sample* or an (executable) *test case* demonstrating the expected behaviour that is not occurring.
+- If you're unable to find an (open) issue addressing the problem, open a new one by using the following [template to report a bug]. Be sure to include:
```

Comment:
```suggestion
- If you're unable to find an issue addressing the problem, open a new one by using the following [template to report a bug]. Be sure to include:
```

Reply:
A bit unclear to use parenthesis (open) here. Better to change to either `an issue` (as I suggested) or `an open issue`.

Reply:
Better without the `()`.